### PR TITLE
Add Consensus transactions tab to account and validator details

### PR DESCRIPTION
**Pull request by buberdds**

### CODE REVIEW

File: src/app/pages/ConsensusAccountDetailsPage/index.tsx [hunk 1]
```diff
@@ -12,18 +13,30 @@ import { RoundedBalance } from '../..//components/RoundedBalance'
...
 import { useRequiredScopeParam } from '../../hooks/useScopeParam'
 import { ConsensusAccountDetailsCard } from './ConsensusAccountDetailsCard'
+import { RouterTabs } from '../../components/RouterTabs'
+
+export type ConsensusAccountDetailsContext = {
+  scope: SearchScope
+  address: string
+}
+
+export const useConsensusAccountDetailsProps = () => useOutletContext<ConsensusAccountDetailsContext>()
```
Comment (lukaw3d) [inline src/app/pages/ConsensusAccountDetailsPage/index.tsx:23]: this breaks hotreloading

Comment (buberdds) [reply src/app/pages/ConsensusAccountDetailsPage/index.tsx]: fixed

File: src/app/pages/ValidatorDetailsPage/index.tsx [hunk 2]
```diff
@@ -14,25 +17,35 @@ import { ValidatorCommission } from '../../components/Validators/ValidatorCommis
...
-import { useLoaderData } from 'react-router-dom'
 import { AddressLoaderData } from 'app/utils/route-utils'
 import { ValidatorSnapshot } from './ValidatorSnapshot'
 
+export type ValidatorDetailsContext = {
+  scope: SearchScope
+  address: string
+}
+
+export const useValidatorDetailsProps = () => useOutletContext<ValidatorDetailsContext>()
```
Comment (lukaw3d) [inline src/app/pages/ValidatorDetailsPage/index.tsx:28]: This probably still breaks hotreload (but i didn't check. entityId API annoying)

**Review by lukaw3d — APPROVED** [review]

File: src/app/pages/ValidatorDetailsPage/index.tsx (hunk 2 above)
Comment (buberdds) [reply src/app/pages/ValidatorDetailsPage/index.tsx]: ah true, fixed

**Pull request merged**